src/assets/scss/assets/components/_menu.scss

```diff
@@ -0,0 +1,167 @@
+@import '../variables/colors';
+@import '../variables/others';
+
+.menu {
```

Comment:
Icons in the menu are not displayed
![Screenshot 2022-05-04 180327](https://user-images.githubusercontent.com/56841893/166710982-1b8d5385-d22a-4cc6-a38a-4a551ca62c8c.jpg)